### PR TITLE
week 5

### PR DESCRIPTION
**Pull request by alenamedved**

### CODE REVIEW

File: 03-task-manager/starter/quizeAnswers.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+1 middleware functions have access to the objects req and res and also the next function. 
+we can modify req or res using middleware, that makes it a powerfull tool. 
+next function calling for the next function - always must end our middleware with next()
```
Comment (Indra12345) [inline 03-task-manager/starter/quizeAnswers.txt:3]: Both true statement and most important ones. Good work. Middlewares are very important in node application and there are many other properties that you will learn las you learn from the documents and come across ideas when you work on project and other parts of the courses such authentication and authorization, testing etc. Keep up the good work. Good work ALena!

File: 03-task-manager/starter/quizeAnswers.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+2CRUD stands for Create Read Update Delete/Destroy
+
+3   1. when using node.js need to install 'dotenv' package and add require('dotenv').config() in the file where we need access to .env - it gives access to .env file
```
Comment (Indra12345) [inline 03-task-manager/starter/quizeAnswers.txt:7]: require('dotenv').config() will attach the object created from key and values of the .env file from .env file to process module's env object and hence you can access that value like process.env.PORT or process.env.password etc. Porcess is a file . . try it to print out console..log(process). It is jumbo file/module and one of the key in that module is env. You guessed it correctly. Just for fun...

File: 03-task-manager/starter/quizeAnswers.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    2. to protect the content of .env file we add it to .gitignore file so it wont be pushed to GitHub
```
Comment (Indra12345) [inline 03-task-manager/starter/quizeAnswers.txt:8]: Excellent work.

File: 03-task-manager/starter/quizeAnswers.txt (hunk 2 above)
Comment (alenamedved) [reply 03-task-manager/starter/quizeAnswers.txt:7]: thanks for the clarification @Indra12345  will do!

**Pull request merged**